package.json

```diff
@@ -5,7 +5,7 @@
   "description": "The MetaMask Design System monorepo",
   "repository": {
     "type": "git",
-    "url": "https://github.com/MetaMask/metamask-design-system.git"
+    "url": "git+https://github.com/MetaMask/metamask-design-system.git"
```

Comment:
Not sure why `yarn constraints` is complaining here this should be prefixed with `git+` I don't think